build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -61,6 +67,9 @@
       <ProjectModelProjectJsonVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch)-rc4-$(CommitCount)</ProjectModelProjectJsonVersion>
       <DependencyModelAndInternalAbstractionsProjectJsonVersion>$(VersionMajor).$(VersionMinor).1-beta-$(CommitCount)</DependencyModelAndInternalAbstractionsProjectJsonVersion>
 
+      <CoreCLRVersion>1.0.2</CoreCLRVersion>
```

Comment:
I think these should go in build.proj with the rest of our hard-coded dependency versions.  They should all be together in 1 spot (for easier update-dependencies), and there is no reason for them to be in "Init".